pkgs/development/python-modules/ansible-lint/default.nix

```diff
@@ -13,27 +13,25 @@
 , wcmatch
 , enrich
 , python
+, tenacity
+, flaky
+, yamllint
```

Comment:
Bonus point: Appending to the bottom is fine, but sorting them, for example in the order they're used in and alphabetically, is helpful when browsing the list.